### PR TITLE
Cluster for demo application

### PR DESCRIPTION
**Pull request by Evreke**
cluster credentials for three profiles:
* embedded-postgres - single node with testcontainers
* embedded-cluster - two node cluster with testcontainers
* external-cluster - two node cluster with externalized configuration

To test this code you have to publish 0.9.2-SNAPSHOT of pg-index-health-testing from https://github.com/mfvanek/pg-index-health/tree/feature/externalize_params

Looking forward to your participation

https://github.com/mfvanek/pg-index-health-spring-boot-demo/issues/11

### CODE REVIEW

File: src/main/java/io/github/mfvanek/pg/index/health/demo/config/EmbeddedPostgresConfig.java [hunk 1]
```diff
@@ -24,7 +25,8 @@
 import javax.sql.DataSource;
 
 @Configuration(proxyBeanMethods = false)
-public class DatabaseConfig {
+@Profile("embedded-postgres")
+public class EmbeddedPostgresConfig {
```
Comment (mfvanek) [inline src/main/java/io/github/mfvanek/pg/index/health/demo/config/EmbeddedPostgresConfig.java:29]: It's not Embedded

File: src/main/java/io/github/mfvanek/pg/index/health/demo/config/EmbeddedPostgresCredentials.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import io.github.mfvanek.pg.connection.ConnectionCredentials;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import javax.annotation.Nonnull;
+
+@Configuration
+public class EmbeddedPostgresCredentials {
```
Comment (mfvanek) [inline src/main/java/io/github/mfvanek/pg/index/health/demo/config/EmbeddedPostgresCredentials.java:19]: It's not a good name. It's not `embedded`

**Review by mfvanek — CHANGES_REQUESTED** [review]

**Comment by Evreke** [comment]
I change profile names and class names according to it. Starter guide added to readme.md

**Comment by Evreke** [comment]
Problem is that now we have to run tests for each profile (probably excluding external-cluster-postgres). 
We can do it with gradle.
Another problem with tests for external-cluster-postgres against external cluster. We have to exclude it's configuration from coverage or somehow run tests with that profile.

**Review by mfvanek — CHANGES_REQUESTED** [review]

**Comment by mfvanek** [comment]
Please update PR and resolve conflicts with master branch. I've migrated to Kotlin DSL

**Pull request closed without merging**